spec/features/job_seekers_can_subscribe_to_a_job_alert_spec.rb

```diff
@@ -45,7 +45,7 @@
         fill_in 'subscription[email]', with: 'jane.doe@example.com'
         click_on 'Subscribe'
 
-        expect(page).to have_content(I18n.t('subscriptions.confirmation.header'))
+        expect(page.html).to include(I18n.t('subscriptions.confirmation.header'))
```

Comment:
Why did these need to change?  The comit messages didn't provide any clarity for me.

Reply:
This was also part of the commit Stuart authored: (adcc3e0) and is possibly related to the previous change that had HTML in the subscription header (also in the same commit). 
Have tested this without using `page.html` and `include()` in the `expect` block and all seems well.